common/options.hpp

```diff
@@ -10,6 +10,10 @@ struct RunOptions {
   size_t iterations = 1;
   std::string root_path;
   std::string report_path;
+
+  // For groupby
+  size_t groups_count = 1;
+  size_t executors = 1;
```

Comment:
Looks like a bad decision to me. Better create a separate structure for a specific dwarf that has required fields (i.e. inherit from run options).